src/client/containers/ProfilePage/ProfilePage.Container.js

```diff
@@ -1,10 +1,61 @@
 import React from 'react';
 import './ProfilePage.Style.css';
+import ContactForm from '../../components/ContactForm/ContactForm.component.js';
+import Purchases from '../../components/Purchases/Purchases.component.js';
+import { useFetchApi } from '../../hooks/UseFetchApi';
+import { useFirebase } from '../../firebase/FirebaseContext';
+import Loader from '../../components/Loader/Loader.component';
+import DeliveryInfoV2 from '../../components/DeliveryInfo/DeliveryInfoV2.component';
+
+const ProfilePageContainer = ({ isAuthenticated }) => {
+  const { auth } = useFirebase();
+  console.log(auth);
```

Comment:
Saru, it is better to remove console.log.

Reply:
👍🏻